userguide/image_classifier/README.md

```diff
@@ -30,6 +30,43 @@ data.save('cats-dogs.sframe')
 data.explore()
 ```
 
+#### Annotating Data
+
+If you only have images without corresponding annotations, you can 
+use the annoation utility built into the image_classifier. An example
+of its usage is shown below with the **PetImages** subfolder above.
+
+```python
+import turicreate as tc
+
+# Load images (Note:'Not a JPEG file' errors are simply warnings, meaning those files will simply be skipped)
+data = tc.image_analysis.load_images('PetImages', with_path=True)
+
+# Use the Annotation GUI to annotate your data.
+annotated_data = tc.image_classifier.annotate(data)
+
+```
+
+If you forget to assign the output of your annotation to a variable, 
+we've included a method to help you recover those annotations. The code
+for that is shown below:
```

Comment:
Again, let's skip the code that's duplicated from the above.

Reply:
Is it alright to duplicate if we move it to the advanced section, or do we assume they've already done it before?